L/libjulia/common.jl

```diff
@@ -335,7 +335,11 @@ function build_julia(ARGS, version::VersionNumber)
         error("Unsupported Julia version")
     end
 
-    julia_compat = version ≥ v"1.7" ? "1.6" : "1.0"
+    julia_compat = version ≥ v"1.6" ? "1.6" : "1.0"
+
+    if version == v"1.5.4"
+        julia_compat = "~1.5.4"
```

Comment:
Why?

Reply:
Indeed not needed, given your explanation below. I removed the 1.5.4 compat.